generators/entity/templates/server/src/test/java/package/web/rest/_EntityResourceIntTest.java

```diff
@@ -17,22 +17,22 @@
  limitations under the License.
 -%>
 package <%=packageName%>.web.rest;
-<% if (databaseType == 'cassandra') { %>
+<% if (databaseType === 'cassandra') { %>
 import <%=packageName%>.AbstractCassandraTest;<% } %>
 import <%=packageName%>.<%= mainClass %>;
-<% if (authenticationType == 'uaa') { %>
+<% if (authenticationType === 'uaa') { %>
 import <%=packageName%>.config.SecurityBeanOverrideConfiguration;
 <% } %>
 import <%=packageName%>.domain.<%= entityClass %>;
 <%_ for (idx in relationships) { // import entities in required relationships
         const relationshipValidate = relationships[idx].relationshipValidate;
         const otherEntityNameCapitalized = relationships[idx].otherEntityNameCapitalized;
-        if (relationshipValidate != null && relationshipValidate === true) { _%>
+        if (relationshipValidate !== null && relationshipValidate === true) { _%>
```

Comment:
I think we can remove the `=== true`, but same comment: it should be done in another PR, it would be simplier